src/main/java/seedu/address/ui/CompanyCard.java

```diff
@@ -59,8 +57,24 @@ public CompanyCard(Company company, int displayedIndex) {
         deadline.setText(company.getDeadline().toString());
         status.setText(company.getStatus().getDescription());
 
-        company.getTags().stream()
-                .sorted(Comparator.comparing(tag -> tag.tagName))
-                .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        if (!company.getPriority().priority.equals("NONE")) {
+            FlowPane priorityPane = new FlowPane();
+
+            Label priorityValue = new Label(company.getPriority().priority);
+
+            if ("HIGH".equals(company.getPriority().priority)) {
+                priorityValue.setStyle("-fx-background-color: #990000;"); // Dark red
+            } else if ("MEDIUM".equals(company.getPriority().priority)) {
+                priorityValue.setStyle("-fx-background-color: #FF8000;"); // Dark orange
+            } else if ("LOW".equals(company.getPriority().priority)) {
+                priorityValue.setStyle("-fx-background-color: #006400;"); // Dark green
+            } else {
+                priorityValue.setStyle("-fx-background-color: #444444;"); // Dark gray (default)
+            }
```

Comment:
Could we abstract out the if-else statements, so that CompanyCard method won't be too long? Might want to consider using switch-case statements to make it more succinct. 

Similarly for CompanyDetailCard

Reply:
done